packages/infra/lib/api-stack.ts

```diff
@@ -565,7 +565,7 @@ export class APIStack extends Stack {
       fhirToBundleCountLambda,
       rateLimitTable,
       searchIngestionQueue: ccdaSearchQueue,
-      searchEndpoint: ccdaSearchDomain.domainEndpoint,
+      searchEndpoint: "https://" + ccdaSearchDomain.domainEndpoint,
```

Comment:
Sorry but this is an intermediary step, the final version is a [downstream PR](https://github.com/metriport/metriport/pull/3857/files#diff-7571b9967bd516636b0e0588f10692b823fb5851591895115c385fec5920fb83R142), where the CCDA search connector returns a full URL and clients don't need to worry about this